src/config/detector-hints.config

```diff
@@ -573,6 +585,17 @@ NO DARK THEME
 
 ================================
 
+shadow.tun.webredirect.org
+shadowobj.eu.org
```

Comment:
I could merge it as it is but as this is the same page but with different URL address then I would add this two URLS under 
`objshadow.tun.webredirect.org` alias and then remove this section as it is duplicate to the earlier one.